website/cue/reference/services/http_scrape.cue

```diff
@@ -2,7 +2,7 @@ package metadata
 
 services: http_scrape: {
 	name:     "HTTP scrape"
-	thing:    "a \(name)"
+	thing:    "an \(name)er"
```

Comment:
Wouldn't this render as `an HTTP scrapeer`?

Reply:
Ah yes, good catch 👍